CMakeLists.txt

```diff
@@ -393,7 +393,12 @@ ELSE(CMAKE_TOOLCHAIN_FILE)
     MESSAGE(WARNING "No compiler identified")
   ENDIF()
   MESSAGE(STATUS "Compiler: ${COMPILER}")
-  MESSAGE(STATUS "Compiler Version: ${CMAKE_CXX_COMPILER_VERSION}")
+  # Workaround for not properly identified CXX compiler in case of IBM XL compiler
+  IF( ${COMPILER} MATCHES "IBM" )
+    MESSAGE(STATUS "Compiler Version: ${CMAKE_C_COMPILER_VERSION}")
+  ELSE()
+    MESSAGE(STATUS "Compiler Version: ${CMAKE_CXX_COMPILER_VERSION}")
+  ENDIF()
 
```

Comment:
On a Cray, I got
-- The C compiler identification is Intel 17.0.0.20170213
-- The CXX compiler identification is Intel 17.0.0.20170213
and later
-- Compiler: Intel
-- Compiler Version: 17.0.0.20170213
It is basically redundant info and better just remove.